cmapPy/math/modz.py

```diff
@@ -0,0 +1,68 @@
+'''
+modz.py
+
+Given a matrix of profiles on which you want to calculate a weighted average, returns a single profile of modz values
+'''
+
+import pandas as pd
+import numpy as np
+import os
+import math
+
+def upper_triangle(correlation_matrix):
+    '''
+    :param correlation_matrix (pandas df): Correlations between all replicates
```

Comment:
Not familiar with this way of formatting arguments; for compatibility with ReadTheDocs, I would restructure/reorganize these so it's the same as the rest of the repo. For example, this one would be: 

Args:
        correlation_matrix (pandas DataFrame): A matrix of correlations between all replicate samples. 

If you need further examples, this is based off the Google style guide.

Reply:
Not going to comment on all your docstrings, but this applies more generally to those as well!

Reply:
Huh, Lev mentioned this and I switched my default to google style, I wonder if it didn't work. Is the only difference that it doesn't have param?

Reply:
Also could be good to clarify why this is necessary as a method separate from np.triu or something similar.